src/lib/import-export/import/importers/importer.ts

```diff
@@ -156,6 +157,30 @@ abstract class BaseBackupImporter extends BaseImporter {
 		await shell.trashItem( dbPath );
 	}
 
+	protected async moveExistingWpContentToTrash( rootPath: string ): Promise< void > {
+		const wpContentDir = path.join( rootPath, 'wp-content' );
+		try {
+			if ( ! fs.existsSync( wpContentDir ) ) {
+				return;
+			}
+			const contentToKeep = [ 'mu-plugins', 'database', 'db.php' ];
+
+			const contents = await fsPromises.readdir( wpContentDir );
+
+			for ( const content of contents ) {
+				const contentPath = path.join( wpContentDir, content );
+
+				if ( contentToKeep.includes( content ) ) {
+					continue;
+				}
+
+				await shell.trashItem( contentPath );
+			}
+		} catch {
+			return;
```

Comment:
Should we perhaps handle the error?

Reply:
We do that in other place as well.
Not handling it, it's not a big deal, as things will keep working, but the user will end up having a "merged" site instead of a completely fresh one.
I think it's ok!
What do you think?

Reply:
In that case, lets keep it that way.